variation/validators/validator.py

```diff
@@ -1,473 +1,183 @@
-            tokens
-        :param List results: Stores validation result objects
-        :param List gene_tokens: List of GeneMatchTokens for a classification
-        :param Dict mane_data_found: MANE Transcript information found
-        :param bool is_identifier: `True` if identifier is given for exact
-            location. `False` otherwise.
-        :param HGVSDupDelModeEnum hgvs_dup_del_mode: Must be: `default`,
-            `copy_number_count`, `copy_number_change`, `repeated_seq_expr`,
-            `literal_seq_expr`. This parameter determines how to represent HGVS dup/del
-            expressions as VRS objects.
-        :param Optional[Endpoint] endpoint_name: Then name of the endpoint being used
-        :param Optional[int] baseline_copies: Baseline copies number
-        :param Optional[CopyChange] copy_change: The copy change
-        :param bool do_liftover: Whether or not to liftover to GRCh38 assembly
-        """
-        raise NotImplementedError
-
-    async def validate(
-            self, classification: Classification,
-            hgvs_dup_del_mode: HGVSDupDelModeEnum = HGVSDupDelModeEnum.DEFAULT,
-            endpoint_name: Optional[Endpoint] = None,
-            baseline_copies: Optional[int] = None,
-            copy_change: Optional[CopyChange] = None,
-            do_liftover: bool = False
+        self, classification: Classification, accessions: List
     ) -> List[ValidationResult]:
-        """Return validation result for a given classification.
-
-        :param Classification classification: A classification for a list of
-            tokens
-        :param HGVSDupDelModeEnum hgvs_dup_del_mode: Must be: `default`,
-            `copy_number_count`, `copy_number_change`, `repeated_seq_expr`,
-            `literal_seq_expr`. This parameter determines how to represent HGVS dup/del
-            expressions as VRS objects.
-        :param Optional[Endpoint] endpoint_name: Then name of the endpoint being used
-        :param Optional[int] baseline_copies: Baseline copies number
-        :param Optional[CopyChange] copy_change: The copy change
-        :param bool do_liftover: Whether or not to liftover to GRCh38 assembly
-        :return: List of ValidationResult's containing valid and invalid
-            results
+        """Get list of validation results for a given classification and accessions
+
+        :param classification: A classification for a list of tokens
+        :param accessions: A list of accessions for a classification
+        :return: List of validation results containing invalid and valid results
         """
-        results = list()
-        errors = list()
 
-        classification_tokens = self.get_classification_tokens(classification)
-        if len(classification.non_matching_tokens) > 0:
-            errors.append(f"Non matching tokens found for "
-                          f"{self.variation_name()}.")
+    async def validate(self, classification: Classification) -> ValidationResult:
```

Comment:
I think this return annotation should be List[ValidationResult]

Reply:
Yes, I probably missed updating other return types